datafusion/expr/src/udf.rs

```diff
@@ -319,7 +319,7 @@ impl ScalarUDF {
 
 impl<F> From<F> for ScalarUDF
 where
-    F: ScalarUDFImpl + Send + Sync + 'static,
```

Comment:
I double checked and indeed `ScalarUDFImpl` already requires `Send + Sync`

https://docs.rs/datafusion/latest/datafusion/logical_expr/trait.ScalarUDFImpl.html